src/dream.mli

```diff
@@ -1387,7 +1387,14 @@ val session_label : request -> string
 val session_expires_at : request -> float
 (** Time at which the session will expire. *)
 
+(** {1 Flash Messages} *)
 
+val flash_messages : ?lifetime:float -> middleware
+type level = Debug | Info | Success | Warning | Error
+
+val clear_flash : level -> request -> unit
```

Comment:
Is this necessary? Shouldn't the middleware now clear the older flash messages?

Reply:
Yes, by default the middleware will clear an old flash message on the next request. I thought `clear_flash` might be useful if one needs a way to get rid of a message created earlier in the process of handling a request (before data is written out to a cookie).

It might be better to just omit this until someone asks for it. I see that Flask does not have a function like this, for example.